rusk/src/lib/services/state.rs

```diff
@@ -329,14 +326,31 @@ impl State for Rusk {
         info!("Received Revert request");
 
         let mut state = self.state()?;
-
         state.revert();
-        self.persist(&mut state)?;
 
         let state_root = state.root().to_vec();
         Ok(Response::new(RevertResponse { state_root }))
     }
 
+    async fn persist(
+        &self,
+        request: Request<PersistRequest>,
+    ) -> Result<Response<PersistResponse>, Status> {
+        info!("Received Persist request");
+
+        let request = request.into_inner();
+
+        let mut state = self.state()?;
+
```

Comment:
Can you add the rusk state root? (Useful to debug imho)

Reply:
What do you mean? Could you clarify?

Reply:
You force pushed in the meanwhile and my comment went to the wrong line